db/src/urls.rs

```diff
@@ -160,14 +189,15 @@ async fn store_raw_html_in_page_snapshot(
     .fetch_one(pool)
     .await?;
 
-    let markdown_result = store_markdown(pool, result.page_snapshot_id, &cleaned_html).await?;
-    println!("Markdown result: {markdown_result:?}");
+    store_markdown(pool, result.page_snapshot_id, &cleaned_html).await?;
+    let markdown_content = html2md::parse_html(&cleaned_html);
+    generate_and_store_summary(pool, result.page_snapshot_id, &markdown_content).await?;
```

Comment:
calling the `generate_and_store_summary` to generate a summary related to `page_snapshot_id`